### ISSUE TITLE
Huawei Mate 9 Mono Maximum Resolution

### ISSUE THREAD

**Issue by tmalone14** [KillerInk/FreeDcam#189]
When I select the mono camera in settings, then change the resolution of the "secondary size" to "5120x3840", the screen starts to flash. I can then get back to the camera interface, but the preview is blank. The screen still flashes. I can go back into settings, but I am unable to change any of the settings while the screen is flashing. The only way to get back to a working state is to clear the app data.

**To Reproduce**
1. Go to "settings"
2. select the mono camera
3. click on "secondary size"
4. select the "5120x3840" entry or the "5120x2880" entry

**Expected behavior**
It should change the resolution and continue working without flashing the screen. All other resolution options seem to work.

**Smartphone (please complete the following information):**
 - Device: Huawei Mate 9
 - OS: Android 8
 - Version: 4.3.39 (the latest version that doesn't crash on startup for me)

**Comment by KillerInk** [comment]
yeah camera is crashing and restarting over and over. thats what you notice as flashing screen.
all huawei stuff added is based on p9. with p10 i think, the secondary size got added. there the mono and color cam had different sizes.

im going to remove that option. think it was never working.
